src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/sentencing/SentencingAdjustmentsNomisApiService.kt

```diff
@@ -15,4 +25,70 @@ class SentencingAdjustmentsNomisApiService(@Qualifier("nomisApiWebClient") priva
     )
     .retrieve()
     .awaitBody()
+
+  suspend fun getSentenceAdjustment(
+    nomisSentenceAdjustmentId: Long,
+  ): NomisAdjustment? = webClient.get()
+    .uri("/sentence-adjustments/{nomisSentenceAdjustmentId}", nomisSentenceAdjustmentId)
+    .retrieve()
+    .awaitBodyOrNullWhenNotFound()
+
+  suspend fun getKeyDateAdjustment(
+    nomisKeyDateAdjustmentId: Long,
+  ): NomisAdjustment? = webClient.get()
+    .uri("/key-date-adjustments/{nomisKeyDateAdjustmentId}", nomisKeyDateAdjustmentId)
+    .retrieve()
+    .awaitBodyOrNullWhenNotFound()
+
+  suspend fun getSentencingAdjustmentIds(
```

Comment:
New (well actually re-added from old commit that removed the migration endpoints)